tests/gocase/unit/info/info_test.go

```diff
@@ -88,4 +88,8 @@ func TestInfo(t *testing.T) {
 		require.GreaterOrEqual(t, lastBgsaveTimeSec, 0)
 		require.Less(t, lastBgsaveTimeSec, 3)
 	})
+
+	t.Run("get cluster information by INFO", func(t *testing.T) {
+		require.Equal(t, "0", util.FindInfoEntry(rdb, "cluster_enabled", "cluster"))
+	})
```

Comment:
@enjoy-binbin Thank you for your contribution.
LGTM.
Additionally, you can add a test for the case when a cluster mode is enabled like the following:
```

t.Run("get cluster information by INFO - cluster not enabled", func(t *testing.T) { // <---- your test case
	require.Equal(t, "0", util.FindInfoEntry(rdb, "cluster_enabled", "cluster"))
})

srv0 := util.StartServer(t, map[string]string{"cluster-enabled": "yes"})
defer func() { srv0.Close() }()
rdb0 := srv0.NewClient()
defer func() { require.NoError(t, rdb0.Close()) }()

t.Run("get cluster information by INFO - cluster enabled", func(t *testing.T) {
	require.Equal(t, "1", util.FindInfoEntry(rdb0, "cluster_enabled", "cluster"))
})

```

Reply:
thanks, i wanted to add one, but it was too late yesterday. took your suggestions